libraries/integration/Microsoft.Bot.Builder.Integration.Runtime/Microsoft.Bot.Builder.Integration.Runtime.csproj

```diff
@@ -41,8 +41,6 @@
     <PackageReference Include="Microsoft.Bot.Builder.Integration.AspNet.Core" Condition=" '$(ReleasePackageVersion)' != '' " Version="$(ReleasePackageVersion)" />
     <PackageReference Include="Microsoft.Bot.Builder.Dialogs.Adaptive.Teams" Condition=" '$(PreviewPackageVersion)' == '' " Version="$(LocalPackageVersion)" />
```

Comment:
Please remove the .Teams package and add the bf-component tag to Microsoft.Bot.Builder.Dialogs.Adaptive.Teams.csproj

<PackageTags>bf-component</PackageTags>

Reply:
Will do.